src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -31,14 +31,20 @@
 class MozLintIssue(Issue):
     def __init__(self, path, column, level, lineno, linter, message, rule, **kwargs):
         self.nb_lines = 1
-        self.path = path
         self.column = column
         self.level = level
         self.line = int(lineno)  # mozlint sometimes produce strings here
         self.linter = linter
         self.message = message
         self.rule = rule
 
+        # Check path is always relative to the repository
+        self.path = path
+        if self.path.startswith(settings.repo_dir):
```

Comment:
Nit: you could use https://docs.python.org/3/library/os.path.html#os.path.relpath

Reply:
Didn't know this one, thanks.